benchmarking/nursery/visualize_plotly/interactive_plots.py

```diff
@@ -28,27 +27,30 @@
 from syne_tune.experiments import load_experiment
 
 
-def plot_tuner_results(tuner_name: str,
-                      experiment_name: str,
-                      return_df: bool = False,
-                      refresh: bool = False,
-                      refresh_rate: int = 60,
-                      height: Optional[int] = None,
-                      width: Optional[int] = None,
-                      max_rows: int = 100,
-                      max_cols: int = 100,
-                      display_dataframe: bool = True,
-                      display_metadata: bool = False,
-                      dimensions_max_cardinality: int = 50,
-                      save_figure: bool = False,
-    ) -> Optional[pd.DataFrame]:
+def plot_tuner_results(
+    tuner_name: str,
+    experiment_name: str,
+    overwrite=True,
```

Comment:
Could you add the type and also makes this one switched off by default?
```suggestion
    overwrite: bool = False,
```
(to make it less likely that a user runs plot_tuner_results("./") which would wipe out all experiments)

Reply:
Oh yep sure, done!